lnbits/helpers.py

```diff
@@ -30,6 +30,10 @@ def url_for(endpoint: str, external: Optional[bool] = False, **params: Any) -> s
 def template_renderer(additional_folders: Optional[List] = None) -> Jinja2Templates:
     folders = ["lnbits/templates", "lnbits/core/templates"]
     if additional_folders:
+        additional_folders += [
+            Path(settings.lnbits_extensions_path, "extensions", f)
```

Comment:
Maybe we want to perform this hack?
```suggestion
            Path(settings.lnbits_extensions_path, "extensions", f[18:] if f.startswith("lnbits/extensions/" else f)
```